doc/nrf/app_dev/device_guides/nrf54l/cryptography.rst

```diff
@@ -308,6 +320,47 @@ You might encounter the following KMU-specific error codes when storing keys in
 * ``PSA_ERROR_ALREADY_EXIST``: One of the required key slots has already been provisioned.
 * ``PSA_ERROR_NOT_SUPPORTED``: Unsupported key type.
 
+.. _ug_nrf54l_crypto_kmu_metadata:
+
+KMU metadata
```

Comment:
This is a lot of low-level detail for this doc section that was originally about API usage.
Maybe these low-level details should only be in the kmu_provision.rst doc? I suspect they are mostly only interesting for users that want to manually provision keys in the KMU